### PR TITLE
champions v3

### PR DESCRIPTION
**Pull request by msarsha**
Hey,

Migrated the champions api to v3

Had to change the tests also - tests checked for 10 f2p champions which is no longer correct.

BTW, That's my first PR.

### CODE REVIEW

File: RiotSharp.Test/RiotApiTest.cs [hunk 1]
```diff
@@ -133,12 +133,12 @@ public void GetChampions_FreeToPlay_Test()
                 var champions = api.GetChampions(RiotApiTestBase.summoner1and2Region, true);
```
Comment (JanOuborny) [inline RiotSharp.Test/RiotApiTest.cs]: Can you add the [EnsureCredibility method](https://github.com/BenFradet/RiotSharp/blob/develop/RiotSharp.Test/RiotApiTest.cs#L25) here and remove the `Ignore` attribute.

File: RiotSharp.Test/RiotApiTest.cs [hunk 2]
```diff
@@ -147,7 +147,7 @@ public void GetChampionsAsync_FreeToPlay_Test()
                 var champions = api.GetChampionsAsync(RiotApiTestBase.summoner1and2Region, true);
```
Comment (JanOuborny) [inline RiotSharp.Test/RiotApiTest.cs]: Same here.

File: RiotSharp/RiotApi.cs [hunk 3]
```diff
@@ -35,7 +35,7 @@ public class RiotApi : IRiotApi
         private const string RunesRootUrl = "/api/lol/{0}/v1.4/summoner";
         private const string RunesUrl = "/{0}/runes";
 
-        private const string ChampionRootUrl = "/api/lol/{0}/v1.2/champion";
+        private const string ChampionRootUrl = "/lol/platform/v3/champions";
```
Comment (JanOuborny) [inline RiotSharp/RiotApi.cs]: Can you rename this to `ChampionsUrl` so it's consistent with the other v3 urls naming.

**Comment by msarsha** [comment]
Changed the url property name.

The tests are already wrapped with the `EnsureCredibility` method.

**Comment by JanOuborny** [comment]
@msarsha yep, I see it know. Github cut those lines of my view and I overlooked it.

**Pull request merged**